deploy/base/kustomization.yaml

```diff
@@ -2,10 +2,8 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
 resources:
+  - gradle-v0.1.yaml
   - https://github.com/tektoncd/pipeline/releases/download/v0.34.1/release.yaml
-  - https://raw.githubusercontent.com/redhat-appstudio/build-definitions/main/tasks/git-clone.yaml
+  - https://raw.githubusercontent.com/tektoncd/catalog/main/task/git-clone/0.5/git-clone.yaml
```

Comment:
This needs to be changed back before merge.

Reply:
Done.